inc/classes/class_db_mysql.php

```diff
@@ -2,48 +2,30 @@
 
 class db
 {
-    public $mysqli = 0;
     public $link_id = 0;
     public $query_id = 0;
-    public $record   = array();
-    public $print_sql_error;
+    public $record = array();
     public $success = false;
     public $count_query = 0;
     public $errors = '';
     public $errorsFound = 0;
     public $connectfailure = 0;  //0= no error, 1=connection error, 2=database error
     public $QueryArgs = array();
 
-  // Construktor
-    public function __construct()
```

Comment:
As said I don't see a large problem with this. 
Only case we should avoid at all costs is that we leak the call parameters of mysqli_connect() in case initialisation fails. If that doesn't happen, then number 2 is perfectly fine.